pkg/cloudprovider/kubevirt/instances_test.go

```diff
@@ -1,347 +1,347 @@
-		expectedError    error
-	}{
-		{"missingVMI", "", cloudprovider.InstanceNotFound},
-		{"hostname", types.NodeName("nodehostname"), nil},
-		{"nodenohostname", types.NodeName("nodenohostname"), nil},
-		{"node.domainname", types.NodeName("node.domainhostname"), nil},
-		{"node.domainnohostname", types.NodeName("node.domainnohostname"), nil},
-	}
-
-	for _, test := range tests {
-		nodeName, err := i.CurrentNodeName(context.TODO(), test.hostname)
-		if nodeName != test.expectedNodeName {
-			t.Errorf("Expected: %v, got: %v", test.expectedNodeName, nodeName)
-		}
-		if test.expectedError != nil && err != nil && err.Error() != test.expectedError.Error() {
-			t.Errorf("Expected: '%v', got '%v'", test.expectedError, err)
-		}
-	}
-}
-
-func TestInstanceExistsByProviderID(t *testing.T) {
-	i := mockInstances(t, "testInstanceExistsByProviderID")
-	tests := []struct {
-		providerID             string
-		expectedInstanceExists bool
-		expectedError          error
-	}{
-		{"kubevirt://missingVMI", false, nil},
-		{"kubevirt://nodePhaseRunning", true, nil},
-		{"kubevirt://nodePhaseSucceeded", true, nil},
-		{"kubevirt://nodePhaseFailed", true, nil},
-	}
-
-	for _, test := range tests {
-		instanceExists, err := i.InstanceExistsByProviderID(context.TODO(), test.providerID)
-		if instanceExists != test.expectedInstanceExists {
-			t.Errorf("Expected: %v, got: %v", test.expectedInstanceExists, instanceExists)
-		}
-		if test.expectedError != nil && err != nil && err.Error() != test.expectedError.Error() {
-			t.Errorf("Expected: '%v', got '%v'", test.expectedError, err)
-		}
-	}
-}
-
-func TestInstanceShutdownByProviderID(t *testing.T) {
-	i := mockInstances(t, "testInstanceShutdownByProviderID")
-	tests := []struct {
-		providerID               string
-		expectedInstanceShutdown bool
-		expectedError            error
-	}{
-		{"notkubevirt://instance", false, fmt.Errorf("ProviderID \"notkubevirt://instance\" didn't match expected format \"kubevirt://InstanceID\"")},
-		{"kubevirt://missingVMI", false, cloudprovider.InstanceNotFound},
-		{"kubevirt://nodePhaseUnset", false, nil},
-		{"kubevirt://nodePhasePending", false, nil},
-		{"kubevirt://nodePhaseScheduling", false, nil},
-		{"kubevirt://nodePhaseScheduled", false, nil},
-		{"kubevirt://nodePhaseRunning", false, nil},
-		{"kubevirt://nodePhaseSucceeded", true, nil},
-		{"kubevirt://nodePhaseFailed", true, nil},
-		{"kubevirt://nodePhaseUnknown", true, fmt.Errorf("Instance is in unkown state (propably host down)")},
-	}
-
-	for _, test := range tests {
-		instanceShutdown, err := i.InstanceShutdownByProviderID(context.TODO(), test.providerID)
-		if instanceShutdown != test.expectedInstanceShutdown {
-			t.Errorf("Expected: %v, got: %v", test.expectedInstanceShutdown, instanceShutdown)
-		}
-		if test.expectedError != nil && err != nil && err.Error() != test.expectedError.Error() {
-			t.Errorf("Expected: '%v', got '%v'", test.expectedError, err)
-		}
-	}
-}
+// package kubevirt
```

Comment:
Uncomment this for the build to pass.
```suggestion
package kubevirt
```